config.toml

```diff
@@ -107,6 +107,11 @@ version = "0.0"
 # point people to the main doc site.
 url_latest_version = "/"
 
+[[params.versions]]
+version = "latest"
+# url = "./docs" # If chinese document is avaliable
+url = '/docs'
```

Comment:
This version looks better. Does this mean if I want to add docs for v0.7.0, I should add a folder /docs/v0.6.0 to store the old version v0.6.0 and put docs for v0.7.0 under /docs/ and then adjust the index like below?

[[params.versions]]
version = "latest"
url = '/docs'

[[params.versions]]
version = "v0.6.0"
url = '/docs/v0.6.0'

Reply:
OK

Reply:
The other version should be in a separate branch, it will be better.